example/src/main/java/com/stripe/example/module/DependencyHandler.java

```diff
@@ -28,7 +28,7 @@
     private static final String PUBLISHABLE_KEY = "pk_test_6pRNASCoBOKtIshFeQd4XMUh";
 
     private AsyncTaskTokenController mAsyncTaskController;
-    private CardInputView mCardInputView;
+    private CardInputWidget mCardInputWidget;
```

Comment:
Nit: The variable is `mCardInputWidget` in this file but `mCardInputView` in others. Would help to get it consistent.